config/kibana.yml

```diff
@@ -18,6 +18,10 @@
 # default to `true` starting in Kibana 7.0.
 #server.rewriteBasePath: false
 
+# Specifies the public URL at which Kibana is available for end users. If
+# `server.basePath` is configured this URL should end with the same basePath.
+#server.publicBaseUrl: ""
```

Comment:
During my testing, I noticed the value I set here isn't passed down to `core.basePath.publicBaseUrl` in my plugin's setup function. I tried a few values in `kibana.yml` as well as `kibana.dev.yml` without luck.

Reply:
I just tested this without issue. You probably need to run `yarn kbn bootstrap` to rebuild the `@kbn/config` pacakge.

Reply:
Ah, yup, that was it! Thanks